Serializer/BaseSerializerHandler.php

```diff
@@ -35,15 +40,22 @@ public function __construct(ManagerInterface $manager)
     }
 
     /**
+     * @param array $formats
+     */
+    public static function setFormats(array $formats)
```

Comment:
* Why static?
* This should be `final`
* Maybe having a `addFormat` would be great too.